ircd/s_user.c

```diff
@@ -1637,6 +1637,10 @@ change_nick_user_host(struct Client *target_p,	const char *nick, const char *use
 		sendto_common_channels_local_butone(target_p, CLICAP_CHGHOST, NOCAPS,
```

Comment:
Instead of abusing current_serial semantics, this should exclude users with extended-monitor enabled:

```suggestion
		sendto_common_channels_local_butone(target_p, CLICAP_CHGHOST, CLICAP_EXTENDED_MONITOR,
```

Reply:
We still want to sent messages to users sharing a channel and who have the extended-monitor cap but who are not currently monitoring the user.

Reply:
Err, yes.  The correct implementation would end up being more complex than this, my brain just failed to process it correctly.